### PR TITLE
Trigger warning in SessionManager.__del__ if there are sessions left.

### PR DESCRIPTION
**Pull request by masell**

### CODE REVIEW

File: sockjs/session.py [hunk 1]
```diff
@@ -399,5 +400,10 @@ def broadcast(self, message):
                 session.send_frame(blob)
 
     def __del__(self):
-        self.clear()
+        if len(self.sessions):
+            warnings.warn(
+                "Unclosed sessions! "
+                "Please call await clear() before del",
```
Comment (asvetlov) [inline sockjs/session.py]: backticks around `await cliear()` make the message more readable.

File: tests/test_session.py [hunk 2]
```diff
@@ -597,3 +597,28 @@ def test_heartbeat(self, make_manager):
...
+
+        _, sm = make_manager()
+        s1 = make_session('id1')
+        s2 = make_session('id2')
+
+        sm._add(s1)
+        sm._add(s2)
+
+        with pytest.warns(RuntimeWarning) as warning:
+            getattr(sm, '__del__')()
```
Comment (asvetlov) [inline tests/test_session.py]: I believe `del sm` should be enough. `getattr()` usage looks too cryptic for my eyes.

Comment (masell) [reply tests/test_session.py]: Tried that, and `__del__` was not triggered. Infact, in python `__del__` is not guaranteed to be called immediately. But with getattr we call the function without the GC cycle.

File: sockjs/session.py (hunk 1 above)
Comment (masell) [reply sockjs/session.py]: Made better warning message.

**Pull request merged**